python/remote-control-center/README.md

```diff
@@ -0,0 +1,123 @@
+
+The solution is designed to translate prompts on the fly if prompts are requested in a language that is not already in the table.  The translation is then persisted for future use.  If a translation already exists, it will not be overwritten.  This allows you to modify the translation to correct for any translation gaps.
+
+### Static Routing features
+Static routing features allow you to set flags or otherwise define parameters that should be used within a contact flow.  Using static features, you can define logic for different levels of event, define external numbers to forward calls, or provide parameters for queues, prompts, music, etc.
+
+One use case for static routing features revolves around how users get routed based on external parameters.  For example, by default you may set a flag to 0 for day to day operations. You may want to define a flag for 1 to play a message to a user before routing to the main menu to share more information, i.e. holds times may be longer than normal due to high call volume. Similarly, you can define a flag level of 2 to play a message indicating the office is closed before disconnecting the user.
+
+### Language Specific Routing features
+Language specific routing features are used to define features that are language dependent but not meant to be translated.  For example, you can use a language specific routing feature to define queue IDs that support specific languages.  By using a dynamic parameter, you can greatly simplify your contact flows to route users to the appropriate prompts, queues, and agents without duplicating contact flows.
+
+## Initial Deployment
+This solution leverages the AWS Serverless Application Model to deploy the backend infrastructure required to implement the solution.
+
+### Deploy the solution
+1. Clone this directory
+2. Ensure the AWS CLI and AWS SAM are installed locally (or leverage a Cloud9 development environment)
+   1. https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html
+3. Run the following commands
+   1. ` sam build `
+   2. ` sam package --s3-bucket <YOUR_S3_BUCKET> --output-template-file packaged.yaml `
+   3. ` sam deploy --template-file packaged.yaml --stack-name remote-control-center   --capabilities CAPABILITY_IAM  `
+4. Wait while the template deploys the following resources
+   1. ConfigTable - The Amazon DynamoDB table used to store configurations
+   2. HydrateConfigTableLambda - An AWS Lambda function used to populate values into the config table
+   3. RetrieveConfigLambda - An AWS Lambda function to be invoked from within a contact flow
+   4. HydrateConfigTableLambdaRole - An AWS IAM Role for HydrateConfigTableLambda
+   5. RetrieveConfigLambdaRole - An AWS IAM Role for RetrieveConfigLambda
```

Comment:
Can you add a description as to what permissions these IAM roles have?

i.e. "An AWS IAM Role for the HydrateConfigTableLambda that allows the lambda to make reads and writes to DynamoDB"

Reply:
Added verbiage around permissions; appended " that allows the function to read and write to the the ConfigTable as well as translate text through Amazon Translate" to lines.